sound/soc/sof/ipc4-topology.c

```diff
@@ -1462,6 +1464,15 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 
 		msg = &pipeline->msg;
 		msg->primary |= pipeline->mem_usage;
+
+		swidget->instance_id = ida_alloc_max(&ppl_ida, ipc4_data->num_ppl, GFP_KERNEL);
```

Comment:
does this compile? i thought you renamed it to max_num _pipelines?

Reply:
> does this compile? i thought you renamed it to max_num _pipelines?

oops, fixed it now.